readthedocs/templates/projects/environmentvariable_detail.html

```diff
@@ -16,12 +16,39 @@
 {% endblock %}
 
 {% block project_edit_content %}
-
-  <p>
-    {% blocktrans trimmed %}
-      The value of the environment variable is not shown here for security purposes.
-    {% endblocktrans %}
-  </p>
+  <dl>
+    <dt>{% trans "Privacy Level" %}</dt>
+    <dd>
+      {% if environmentvariable.public %}
+        <p><strong>{% trans "Public" %}</strong></p>
+        <p>
+          {% blocktrans trimmed %}
+            The environment variable is available in pull request builds and shown here.
```

Comment:
This is redundant with the message below

```suggestion
            The environment variable is available in pull request builds.
```